landbosse/excelio/XlsxReader.py

```diff
@@ -98,13 +139,19 @@ def read_xlsx_and_fill_defaults(self, input_xlsx, project):
         incomplete_input_dict['road_length_adder_m'] = project['Road length adder (m)']
         incomplete_input_dict['fraction_new_roads'] = project['Percent of roads that will be constructed']
         incomplete_input_dict['road_quality'] = project['Road Quality (0-1)']
+        incomplete_input_dict['site_prep_area_m2'] = project['Site prep area for Distributed wind (m2)']
```

Comment:
This line means that utility-scale only version of LandBOSSE will not crash if it is passed a spreadsheet made only for distributed wind.